ddp-workspace/projects/ddp-dsm-ui/src/app/home/home.component.html

```diff
@@ -1,12 +1,4 @@
-<span *ngIf="!auth.authenticated() && !notAllowedToLogin">
-  <h1>Please login</h1>
-</span>
-
-<span *ngIf="notAllowedToLogin">
-  <h1>You are not allowed to login</h1>
-  <h3>Please contact someone =)</h3>
-</span>
-
 <span *ngIf="auth.authenticated()">
   <h1>Welcome to the DDP Study Management System</h1>
+  <h2 *ngIf="auth.realmList.length > 1">You have selected <b style="color: blue; text-transform: uppercase">{{selectedRealm}}</b> study.</h2>
```

Comment:
suggest avoiding inline styles when possible

Reply:
Right, this component is just temporary, that is why I left it so, but okay, will change it now. Thanks.